app/functional_tests_fixtures/__init__.py

```diff
@@ -472,3 +509,21 @@ def _create_service_inbound_api(service_id, user_id):
     inbound_api.bearer_token = "1234567890"
 
     save_service_inbound_api(inbound_api)
+
+
+def _create_inbound_sms(service, count):
+
+    num_existing = len(dao_get_inbound_sms_for_service(service.id, limit=count))
+
+    for _ in range(num_existing, count):
```

Comment:
Minor - feel free to ignore:
If I understand correctly we are trying to create inbound sms if num_existing is less than count with `range(num_existing, count)`, is that right?

we can do something like `num_to_create = max(0, count - num_existing)` to make it clear how many we’re iterating over and avoids any potential confusion with range. 

Then you could do `range(num_to_create)`